conf/nutch-default.xml

```diff
@@ -1874,6 +1874,72 @@ visit https://wiki.apache.org/nutch/SimilarityScoringFilter-->
   </description>
 </property>
 
+
+<!-- lib-htmlunit plugin properties; applies to protocol-htmlunit -->
+
+<property>
+  <name>htmlunit.page.load.delay</name>
```

Comment:
I can share them but that would mean changing the name of properties defined for selenium which is in short affecting the selenium code as well. Are we okay with that?

Currently, all properties in nutch-default.xml has "selenium" prefix and hence the change is required.